src/sentry/data_export/processors/issues_by_tag.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry import tagstore
+from sentry.models import EventUser, Group, get_group_with_redirect
+
+from .base import ExportProcessor
+from ..base import ExportError
+
+
+class IssuesByTagProcessor(ExportProcessor):
+    """
+    Export processor for data exports of issues data based on a provided tag
+    """
+
+    def __init__(self, project_id, group_id, key, environment_id):
+        self.project = self.get_project(project_id)
+        self.group = self.get_group(group_id, self.project)
+        self.key = key
+        self.environment_id = environment_id
+        self.fields = self.get_fields(self.key)
+        self.lookup_key = self.get_lookup_key(self.key)
+        # Ensure the tag key exists, as it may have been deleted
+        try:
+            tagstore.get_tag_key(self.project.id, environment_id, self.lookup_key)
+        except tagstore.TagKeyNotFound:
+            raise ExportError("Requested key does not exist")
+        self.callbacks = self.get_callbacks(self.key, self.group.project_id)
+
+    @staticmethod
+    def get_group(group_id, project):
+        try:
+            group, _ = get_group_with_redirect(
+                group_id, queryset=Group.objects.filter(project=project)
+            )
+            return group
+        except Group.DoesNotExist:
+            raise ExportError("Requested issue does not exist")
+
+    @staticmethod
+    def get_fields(key):
+        if key == "user":
+            return [
+                "value",
+                "id",
+                "email",
+                "username",
+                "ip_address",
+                "times_seen",
+                "last_seen",
+                "first_seen",
+            ]
+        else:
+            return ["value", "times_seen", "last_seen", "first_seen"]
+
+    @staticmethod
+    def get_lookup_key(key):
+        return six.text_type("sentry:{}".format(key)) if tagstore.is_reserved_key(key) else key
+
+    @staticmethod
+    def get_eventuser_callback(project_id):
+        def attach_eventuser(items):
+            users = EventUser.for_tags(project_id, [i.value for i in items])
+            for item in items:
+                item._eventuser = users.get(item.value)
+
+        return attach_eventuser
+
+    @staticmethod
+    def get_callbacks(key, project_id):
+        return [IssuesByTagProcessor.get_eventuser_callback(project_id)] if key == "user" else []
+
+    @staticmethod
+    def serialize_issue(item, key):
```

Comment:
the `serialize_issue` name is a little confusing for me, to me it looks like this is the tag result and user data? If we want to follow the nomenclature that was used, I think `serialize_row` would be appropriate

Reply:
Renamed it in 0e1e7a0!